src/Components/Article.js

```diff
@@ -19,15 +19,18 @@ class Article extends React.Component {
     return (
       <div>
         <Card.Group>
-          <Card onClick={this.handleClick}>
+          {this.props.source}
+          <Card onClick={this.handleClick} className="cardtest">
```

Comment:
Could you please choose a more meaningful name than "cardtest"?

Reply:
I did